### ISSUE TITLE
🐛 Bug: Search and remove `any` from codebase

### ISSUE THREAD

**Issue by DhairyaMajmudar** [json-schema-org/website#1034]
### Describe the bug

-> While performing a quick VS Code search its found that there are several places in  the codebase where `any` datatype is used,
-> Using `any` in place of correct data type alters the strictness of typescript hence its a not a good practice to use them.

-> The main objective of this issue is to find the files where `any` keyword is used and replace them with the correct data type definition.  

### Steps To Reproduce

Do a VSCode Search to find the files using `any` keyword. They are mostly used in component files

### Expected Behavior

There should be no use of `any` keyword in the codebase  

### Device Information [optional]

```markdown
- OS: Linux (Debian)
- Browser: Firefox
- version: Latest
```

### Are you working on this issue?

No

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

**Comment by Asymtode712** [comment]
@DhairyaMajmudar I would like to work on this issue.....Please assign it to me

**Comment by DhairyaMajmudar** [comment]
> @DhairyaMajmudar I would like to work on this issue.....Please assign it to me

Looking forward for your PR, also at some instances you might need to create a custom data type at that moment pls. take care with the naming, Best wishes 🚀

**Comment by Asymtode712** [comment]
@DhairyaMajmudar I am done with most of the components....I have a doubt: Do I need to remove the `any` keywords from the `pages` directories too....

**Comment by DhairyaMajmudar** [comment]
> @DhairyaMajmudar I am done with most of the components....I have a doubt: Do I need to remove the `any` keywords from the `pages` directory too....

yep, if those files have `any` keyword then pls. remove from there also.

**Comment by idanidan29** [comment]
Hey, @DhairyaMajmudar  I noticed that this issue hasn't been active for a while. Is it still open for contribution? If so, I'd be happy to take it up and get assigned. Let me know

**Comment by Utkarsh-123github** [comment]
Hi @benjagm I'd love to work on this. Could you please assist this to me ?

**Comment by vtushar06** [comment]
Hi @Utkarsh-123github and @benjagm ,
I’d be happy to collaborate on this issue alongsides @Utkarsh-123github , if that’s okay. This seems like a great opportunity to work together and speed up the refactoring process across the codebase. Please let me know, 
Thanks you,
@vtushar06

**Comment by Utkarsh-123github** [comment]
Hi @vtushar06 ,
Thanks a lot for showing interest in collaborating! Currently, this issue is on hold for a little while. Once I resume work on it, we’ll definitely notify you — would be great to work together and speed things up collaboratively.

Appreciate your enthusiasm! 🙌

**Comment by animeshsahoo1** [comment]
@Utkarsh-123github let me know if I can be of some help aswell, would love to collaborate, will wait for updates on the issue

**Comment by vtushar06** [comment]
Thanks @Utkarsh-123github! Sounds great — I’ll stay tuned and would love to collaborate once things resume. 😊

**Comment by vtushar06** [comment]
Hii @Utkarsh-123github, As mentioned in issue #1959, I would start work here, Please assign this issue to me.

**Comment by Adityakumar37** [comment]
@benjagm if this issue is still open i would like to work on this

**Comment by vtushar06** [comment]
Thanks @Adityakumar37 for interest, But as you can see I already raised PR regarding this.

**Comment by manya1632** [comment]
Hi @vtushar06  ! I’m comfortable working with TypeScript and would love to help. If the issue is still in progress, I'd like to work alongside if it fits.

**Comment by vtushar06** [comment]
@manya1632 Thanks , you can share your thoughts on raised PRs, if you find any more changes or implementation needed, we can surely have a discussion on those.
